src/styles/index.scss

```diff
@@ -1,3 +1,145 @@
 body {
   margin: 0;
 }
+
+.product-card {
+  width: 200px;
+  height: auto;
+
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
it is common practice to apply `box-sizing: border-box` for all elements via * selector